src/HashtagHandler.js

```diff
@@ -0,0 +1,60 @@
+var log  = require('npmlog');
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
+
+/**
+ * HashtagHandler - Handler for hashtag room creation and messaging
+ * @class
+ * @extends {external:TwitterHandler}
+ *
+ * @param  {MatrixTwitter}   twitter
+ * @param  {matrix-appservice-bridge.Bridge}   bridge
+ */
+var HashtagHandler = function (bridge, twitter) {
+  TwitterHandler.call(this,bridge,"#","hashtag");
+  this.twitter = twitter;
+}
+
+HashtagHandler.prototype.onRoomCreated = function (alias,entry){
+  console.log(entry);
+  this.twitter.add_hashtag_feed(
+    entry.remote.getId().substr("hashtag_".length),
+    entry
+  );
+}
+
+HashtagHandler.prototype.processMessage = function (event, request, context) {
+    this.twitter.send_matrix_event_as_tweet(event,context.senders.matrix,context.rooms.remote);
```

Comment:
And this is why spaces after commas are so useful. Good luck reading how many arguments this function takes.